SPECS-EXTENDED/python-iniconfig/python-iniconfig.spec

```diff
@@ -0,0 +1,133 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+
+%undefine mariner_module_ldflags
+
+Name:               python-iniconfig
+Version:            1.1.1
+Release:            15%{?dist}
+Summary:            Brain-dead simple parsing of ini files
+# SDPX
+License:            MIT
+URL:                http://github.com/RonnyPfannschmidt/iniconfig
+BuildArch:          noarch
+BuildRequires:      python3-devel
+BuildRequires:      pyproject-rpm-macros
+BuildRequires:      python3-pip
+BuildRequires:      python3-setuptools
+BuildRequires:      python3-setuptools_scm >= 8.0.3-1
+BuildRequires:      python3-six
+BuildRequires:      python3-wheel
+
+# pytest 6+ needs this and this uses pytest for tests
+%bcond_without tests
```

Comment:
Please use the `with_check` macro instead - that's what Azure Linux uses. `tests` is local to the spec and we don't really use the `--with-X` logic.